README.md

```diff
@@ -240,9 +278,15 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Question:** Why did the number of posts not change when you were redirected back to the `/users` route?
 
-* **Your Answer:** Whenever we modify our data with a Create, Update, or Delete, we have a few options on how to make our frontend reflect those changes. What options can you think of?
+* **Your Answer:**
+
+Because state was not updated.
+
+* **Question:** Whenever we modify our data with a Create, Update, or Delete, we have a few options on how to make our frontend reflect those changes. What options can you think of?
+
+* **Your Answer:**
 
-* **Question:**
+refresh state so the app reflects the most recent changes made.
```

Comment:
We could pull the data again and update state but that could be very costly for some larger apps. We could do a targeted pull to only receive the updated data. Or if we get a successful message back from the server we could update the local state to reflect the perceived database changes.